core/tests/protractor_desktop/explorationEnableCorrectnessFeedback.js

```diff
@@ -0,0 +1,142 @@
+    responseEditor.setFeedback(forms.toRichText('Wrong!'));
+    explorationEditorMainTab.moveToState('End');
+    explorationEditorMainTab.setInteraction('EndExploration');
+    // Turn on correctness feedback.
+    applyCommonSettings();
+
+    // Go back to mark the solution as correct.
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorMainTab.moveToState('First');
+    responseEditor = explorationEditorMainTab.getResponseEditor(0);
+    responseEditor.markAsCorrect();
+    expect(tickMark.isDisplayed()).toBe(true);
+    explorationEditorPage.saveChanges();
+  });
+
+  it('should turn on enable correctness feedback before create interaction.' +
+     'Correctness feedback is set in add response modal.', function() {
+    // Turn on correctness feedback first.
+    applyCommonSettings();
+
+    // Go to main tab to create interactions.
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorMainTab.setStateName('First');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'Select the right option.'));
+
+    // Create interaction without close the add response modal. Set
+    // correctness in the modal.
+    explorationEditorMainTab.setInteractionWithoutCloseAddResponse('TextInput');
+    responseEditor = explorationEditorMainTab.getResponseEditor('pop');
+    responseEditor.markAsCorrect();
+
+    // Set the response for this interaction and close it.
+    explorationEditorMainTab.setResponse(
+      'TextInput', forms.toRichText('Correct!'),
+      'End', true, 'Equals', 'One');
+
+    expect(tickMark.isDisplayed()).toBe(true);
+    responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setFeedback(forms.toRichText('Wrong!'));
+    explorationEditorMainTab.moveToState('End');
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorPage.saveChanges();
+  });
+
+  it('should turn on correctness feedback before create interaction.' +
+    'Correctness feedback is set in response editor.', function() {
+    // Turn on correctness feedback first.
+    applyCommonSettings();
+
+    // Go back to main tab to create interactions.
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorMainTab.setStateName('First');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'Select the right option.'));
+    explorationEditorMainTab.setInteraction('NumericInput');
+
+    // Set correctness in response editor.
+    responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.markAsCorrect();
+    responseEditor.setFeedback(forms.toRichText('Correct!'));
+    responseEditor.setDestination('End', true, true);
+    expect(tickMark.isDisplayed()).toBe(true);
+
+    explorationEditorMainTab.moveToState('End');
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorPage.saveChanges();
+  });
```

Comment:
Should we also test the expected behavior of these changes to get reflected in the player mode? (OR that's going to be a separate test?)

Reply:
Added in afterEach